scm-plugins/scm-git-plugin/src/main/java/sonia/scm/repository/spi/SimpleGitWorkdirFactory.java

```diff
@@ -72,7 +73,11 @@ public SimpleGitWorkdirFactory(WorkdirProvider workdirProvider) {
   }
 
   private String createScmTransportProtocolUri(File bareRepository) {
-    return ScmTransportProtocol.NAME + "://" + bareRepository.getAbsolutePath();
+    if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
Please use `sonia.scm.util.SystemUtil`. With the use of platform, it should easier to test. Have a look at [BaseDirectory](https://github.com/scm-manager/scm-manager/blob/dbca56ce6b44629d1fe8422e5fc15bc149ca92c2/scm-core/src/main/java/sonia/scm/BaseDirectory.java) and its test [BaseDirectoryTest](https://github.com/scm-manager/scm-manager/blob/dbca56ce6b44629d1fe8422e5fc15bc149ca92c2/scm-core/src/test/java/sonia/scm/BaseDirectoryTest.java).